src/Cli/dotnet/commands/dotnet-test/TestCommandParser.cs

```diff
@@ -150,11 +151,11 @@ private static Command ConstructCommand()
             command.AddOption(NoLogoOption);
             command.AddOption(ConfigurationOption);
             command.AddOption(FrameworkOption);
-            command.AddOption(CommonOptions.LongFormRuntimeOption.WithHelpDescription(command, LocalizableStrings.RuntimeOptionDescription));
+            command.AddOption(CommonOptions.RuntimeOption.WithHelpDescription(command, LocalizableStrings.RuntimeOptionDescription));
             command.AddOption(NoRestoreOption);
             command.AddOption(CommonOptions.InteractiveMsBuildForwardOption);
             command.AddOption(CommonOptions.VerbosityOption);
-            command.AddOption(CommonOptions.LongFormArchitectureOption);
```

Comment:
Why is this change neede?

Reply:
Because we'll support --arch/--a for architecture and no more --a for test adapter or have I misunderstood?

https://github.com/dotnet/sdk/blob/a5dae2aaee040aea5dedc2c16963360a87585e31/src/Cli/dotnet/CommonOptions.cs#L122